src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs

```diff
@@ -42,7 +42,6 @@ public SearchView(SearchViewModel searchViewModel, DynamoViewModel dynamoViewMod
             viewModel = searchViewModel;
             this.dynamoViewModel = dynamoViewModel;
 
-            DataContext = viewModel;
```

Comment:
DataContext is set in Initialization. Tooltip is bound to TargetUpdated. That means as soon as DataContext is set tooltip tries to show something. But main dynamo window is not loaded yet. And it fails.